src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationModal.js

```diff
@@ -1,42 +1,91 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
 import PropTypes from 'prop-types'
+import { useParams } from 'react-router-dom'
+import { toast } from 'react-toastify'
 import Modal from '../../../generic/Modal/Modal'
 import { IMAGE_CLASSIFICATION_COLORS as COLORS } from '../../../../library/constants/constants'
 import ImageAnnotationModalTable from './ImageAnnotationModalTable'
 import ImageAnnotationModalMap from './ImageAnnotationModalMap'
-import { Footer, Legend, LegendItem, LegendSquare } from './ImageAnnotationModal.styles'
+import {
+  Footer,
+  Legend,
+  LegendItem,
+  LegendSquare,
+  LoadingContainer,
+} from './ImageAnnotationModal.styles'
+import { useDatabaseSwitchboardInstance } from '../../../../App/mermaidData/databaseSwitchboard/DatabaseSwitchboardContext'
+import LoadingIndicator from '../../../LoadingIndicator/LoadingIndicator'
 
-const ImageAnnotationModal = ({ dataToReview, setDataToReview }) => {
+const ImageAnnotationModal = ({ imageId, setImageId }) => {
+  const { databaseSwitchboardInstance } = useDatabaseSwitchboardInstance()
+  const { projectId } = useParams()
+  const [dataToReview, setDataToReview] = useState()
+  const [growthForms, setGrowthForms] = useState()
+  const [benthicAttributes, setBenthicAttributes] = useState()
   const [selectedPoints, setSelectedPoints] = useState([])
   const [highlightedPoints, setHighlightedPoints] = useState([])
 
+  useEffect(() => {
```

Comment:
prefer named `useEffect` functions

Reply:
a pattern Melissa used for this project is adding `_useEffectFunctionName = useEffect(()=> { `.... 

the project is set to ignore vars starting with _ so the eslint warning doesn't go off